website/package.json

```diff
@@ -24,7 +24,7 @@
     "@mdx-js/react": "^3.0.1",
     "@zio.dev/caliban-deriving": "0.0.1",
     "@zio.dev/interop-monix": "2023.1.3-b2b2206862ba",
-    "@zio.dev/izumi-reflect": "2.2.3",
+    "@zio.dev/izumi-reflect": "2024.11.11-425ca6af569c",
```

Comment:
Won't `2.3.10` (latest stable) work as well? Or are there changes in the latest snapshot that we need?

Reply:
The versioning schema for their documentation has changed

Here is the latest version:
https://www.npmjs.com/package/@zio.dev/izumi-reflect/v/2024.11.11-425ca6af569c